### PR TITLE
ENH: adding pf1k0 el3602

### PR DESCRIPTION
**Pull request by joshc-slac**
## Description
Adding ADC for Power Meter OIP

## Motivation and Context
Need ADC to interpret amplifier output

## Pre-merge checklist
- [ ] Code works interactively
- [ ] Code contains descriptive comments
- [ ] Test suite passes locally
- [ ] Libraries are set to fixed versions and not ``Always Newest``
- [ ] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

File: plc-kfe-motion/_Config/IO/Device 1 (EtherCAT)/Power (EK1200)/PLC Junction 2 (EK1122)/X2 SXR EC1 (EK1100)/EC1 E2 (EK1122)/X1 PF1K0-WFS (EK1100)/PF1K0-EL3602.xti
Comment (joshc-slac) [inline plc-kfe-motion/_Config/IO/Device 1 (EtherCAT)/Power (EK1200)/PLC Junction 2 (EK1122)/X2 SXR EC1 (EK1100)/EC1 E2 (EK1122)/X1 PF1K0-WFS (EK1100)/PF1K0-EL3602.xti:1]: I am 95% sure the double inclusion of this file and the inclusion of this file named as Term216 are github artifacts not present on git or if you were to pull this branch with git... will double check that.

Comment (ZLLentz) [reply plc-kfe-motion/_Config/IO/Device 1 (EtherCAT)/Power (EK1200)/PLC Junction 2 (EK1122)/X2 SXR EC1 (EK1100)/EC1 E2 (EK1122)/X1 PF1K0-WFS (EK1100)/PF1K0-EL3602.xti:1]: The files likely exist, but possibly only one variant is referenced in other config files, making them not present in the solution. So only the main one is truly active. In this case it'd be safe (and preferred) to delete the others.

Comment (joshc-slac) [reply plc-kfe-motion/_Config/IO/Device 1 (EtherCAT)/Power (EK1200)/PLC Junction 2 (EK1122)/X2 SXR EC1 (EK1100)/EC1 E2 (EK1122)/X1 PF1K0-WFS (EK1100)/PF1K0-EL3602.xti:1]: Ah you are totally correct

**Review by ghalym — APPROVED** [review]
LGTM!

**Pull request merged**